package NYT.NObjectClient.NProto;

import "yt/ytlib/misc/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqGetId
{ }

message TRspGetId
{
    required NYT.NProto.TGuid object_id = 1;
}

////////////////////////////////////////////////////////////////////////////////

message TReqCheckPermission
{
    required string user = 1;
    required int32 permission = 2;
}

message TRspCheckPermission
{
    required int32 action = 1;
    optional NYT.NProto.TGuid object_id = 2;
    optional string subject = 3;
}

////////////////////////////////////////////////////////////////////////////////